utils/show.cc

```diff
@@ -468,11 +599,28 @@ int tr_main(int argc, char* argv[])
         }
         else
         {
+            bool print_ok_files = false;
+            if (opts.print_files && opts.verify)
+            {
+                /* If opts.verify, verify() will print the file list including the status.
+                 * So disable the file list in showInfo(). */
+                print_ok_files = true;
+                opts.print_files = false;
```

Comment:
Looks to me that `opts.print_files` should be set inside `parseCommandLine()` to be more consistent with the existing code organisation.

Then you can call `verify()` like `verify(opts, opts.print_files && opts.verify, metainfo)`.

Reply:
Or, you can change

https://github.com/transmission/transmission/blob/0f321b850342646c8ca3fd5e54a75220e34865cf/utils/show.cc#L270

to

```cpp
if (opts.print_files && !opts.verify)
```

So that
```cpp
bool verify(app_opts const& opts, bool print_ok_files, tr_torrent_metainfo const& metainfo)
```
can directly check `opts.print_file` instead of `print_ok_files` and become
```cpp
bool verify(app_opts const& opts, tr_torrent_metainfo const& metainfo)
```
instead.